traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -908,14 +778,12 @@ func getServers(h http.Header, params map[string]string, tx *sqlx.Tx, user *auth
 		"dsId":             {Column: "dss.deliveryservice", Checker: nil},
 	}
 
-	if version.Major >= 3 {
-		queryParamsToSQLCols["cachegroupName"] = dbhelpers.WhereColumnInfo{
-			Column:  "cg.name",
-			Checker: nil,
-		}
+	queryParamsToSQLCols["cachegroupName"] = dbhelpers.WhereColumnInfo{
```

Comment:
nit: this could be moved up with the rest of the entries

Reply:
Entry moved up in e1534cc5a6